joomla/joomla-cms
# [4.0] Fix featured articles not shown in frontend

Pull Request for Issue #28628.

### Summary of Changes

Corrects dates in query when saving featured article.

>### Testing Instructions
> Write an article, set is published and featured.
> Make a Featured blog.
> ### Expected result
> 
> The article is visible
> ### Actual result
> 
> The article is not visible in the featured blog, but in a normal categoy blog

### Documentation Changes Required

No.
